File: Lesson/src/lesson2/whileLoop/SubMenuBuilder.java
package lesson2.whileLoop;

import MenuBuilder.ConsoleMenuBuilder;
import lesson2.whileLoop.actions.*;

public class SubMenuBuilder {

    static public ConsoleMenuBuilder getMenu(String prnTitle) {
        return new ConsoleMenuBuilder(prnTitle)
                .AddMenuItem("1", "Sum of input numbers", new InputNumber())
                .AddMenuItem("2", "Input user's name", new InputName())
                .AddMenuItem("0", "Back to previous menu", null, true);
    }
}

File: Lesson/src/lesson14/SubMenuBuilder.kt
package lesson14

import MenuBuilder.ConsoleMenuBuilder
import lesson14.actions.Lesson14

class SubMenuBuilder {
    companion object {
        @JvmStatic
        fun getMenu(prnTitle: String): ConsoleMenuBuilder {
            val LESSON14 = "Lesson14"

            return ConsoleMenuBuilder(prnTitle)
                    .AddMenuItem("1", LESSON14, Lesson14())
                    .AddMenuItem("0", "Go to previous menu", null, true)
        }
    }
}